src/tokenFactory/tokensFactory.ts

```diff
@@ -35,7 +35,12 @@ export function decodeJWT(jwt) {
 }
 
 function signJWT(jwt, rawPrivateKey) {
-  return new _.TokenSigner('ES256K', rawPrivateKey).sign(jwt);
+  if (jwt.header && jwt.payload){
+  	return new _.TokenSigner('ES256K', rawPrivateKey).sign(jwt.payload, false, jwt.header);
```

Comment:
To understand this call, see: https://github.com/blockstack/jsontokens-js/blob/126ef31a5fd0735d01d7ebd5ce781b6305df1e29/src/signer.ts#L67